orchestrator_mapreduce.py

```
@@ -0,0 +1,478 @@
+                        timeout=self.subtask_timeout,
+                    )
+                    subtask.result = result
+                    subtask.status = TaskStatus.COMPLETED
+
+                    # Track outcome without blocking the event loop
+                    try:
+                        await loop.run_in_executor(
+                            None, track_outcome, subtask.protocol, result
+                        )
+                    except Exception:
+                        pass  # Tracking failure shouldn't fail the subtask
```

Ruff (0.15.18)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)